lib/produce/options.rb

```diff
@@ -79,7 +79,22 @@ def self.available_options
                                      verify_block: Proc.new do |value|
                                         ENV["FASTLANE_TEAM_NAME"] = value
                                      end),
-        
+        FastlaneCore::ConfigItem.new(key: :group_name,
+                                     short_option: "-n",
+                                     env_name: "PRODUCE_GROUP_NAME",
+                                     description: "The name of the group that is created",
+                                     optional: true,
+                                     verify_block: Proc.new do |value|
+
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :group_identifier,
+                                     short_option: "-g",
+                                     env_name: "PRODUCE_GROUP_IDENTIFIER",
+                                     description: "The identifier of the group that is created",
```

Comment:
`The identifier of the group that is created`

Does that mean the app will be created within the group with the given identifier? If so, let's change it to something like

`The group identifier the application should be created in`